hooks/afterPrepareHook.js

```diff
@@ -61,7 +61,11 @@ function run(cordovaContext) {
  */
 function activateUniversalLinksInAndroid(cordovaContext, pluginPreferences) {
   // inject preferenes into AndroidManifest.xml
-  androidManifestWriter.writePreferences(cordovaContext, pluginPreferences);
+
```

Comment:
There is an assumption in the hook that says config.xml must reside in the project root -- ours doesn't. We handle updating the keys in our gulpfile so we didn't need this hook.